CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# dbt_quickbooks v0.5.1
+## Bug Fixes 🐛🪛
+- Created indices for `double_entry_transactions` models. Used row_number functions for `payment`, `bill_payment` and `transfer` models.
+- Removed transaction index on final `quickbooks__general_ledger` model, replaced by the newer indices in the sub-ledgers. 
```

Comment:
Would you be able to add this PR as the reference for each each of these bullets.
```suggestion
- Created indices for `double_entry_transactions` models. Used row_number functions for `payment`, `bill_payment` and `transfer` models. ([#41](https://github.com/fivetran/dbt_quickbooks/pull/41))
- Removed transaction index on final `quickbooks__general_ledger` model, replaced by the newer indices in the sub-ledgers. ([#41](https://github.com/fivetran/dbt_quickbooks/pull/41))
```